.github/workflows/ci.yml

```diff
@@ -14,20 +14,15 @@ jobs:
       fail-fast: false
       matrix:
         gemfile:
-          - Gemfile.rails-5.2-stable
-          - Gemfile.rails-6.0-stable
           - Gemfile.rails-6.1-stable
           - Gemfile.rails-7.0-stable
-        ruby-version: ['3.1', '3.0', '2.7']
+          - Gemfile.rails-7.2-stable
+        ruby-version: ['3.1', '3.0']
         exclude:
-          - gemfile: Gemfile.rails-6.0-stable
+          - gemfile: Gemfile.rails-7.2-stable
             ruby-version: "3.0"
-          - gemfile: Gemfile.rails-6.0-stable
-            ruby-version: "3.1"
-          - gemfile: Gemfile.rails-5.2-stable
-            ruby-version: "3.0"
-          - gemfile: Gemfile.rails-5.2-stable
-            ruby-version: "3.1"
+          - gemfile: Gemfile.rails-7.2-stable
+            ruby-version: "2.7"
```

Comment:
Do you need to exclude 2.7 when it is no longer included in ruby-version above?

Reply:
Good catch, I will remove it.